uber/models.py

```diff
@@ -1749,9 +1749,9 @@ class ArbitraryCharge(MagModel):
 
 
 class ApprovedEmail(MagModel):
-    subject = Column(UnicodeText)
+    ident = Column('subject', UnicodeText)  # TODO: rename column to "ident" in the database; will require a db migration
```

Comment:
This change doesn't require a database migration, because we've told SQLAlchemy to continue calling the database column ``subject`` and to merely have the column property called ``ident``.  In the long run we should probably rename the database column to the consistent.  I don't remember exactly where we're at with db migrations at the moment.  I recall seeing some stuff about moving to Alembic but I haven't been keeping up with all of that, so I just put this to maintain compatibility for the moment.